docs/ENVIRONMENT.md

```diff
@@ -21,7 +21,7 @@ This document provides detailed information about environment variables for the
 | `LOG_FILE`                  | Either an output filepath to log to, or 'console'.                                                   | `string` | console                                        |
 | `LOG_LEVEL`                 | Logging level for filtering logs.                                                                    | `string` | warn                                           |
 | `PORT`                      | Port the Gateway server listens on.                                                                  | `number` | 9000                                           |
-| `POSTGRES_AERIE_MERLIN_DB`  | Name of Merlin Postgres database.                                                                    | `string` | aerie_merlin                                   |
+| `POSTGRES_AERIE_DB`         | Name of Aerie Postgres database.                                                                     | `string` | aerie                                          |
```

Comment:
Looks like `POSTGRES_AERIE_DB` isn't actually used anywhere anymore. I'd be happy with either removing this doc and keeping the hardcoded `"aerie"` or actually reading this env var within [`db.ts`](https://github.com/NASA-AMMOS/aerie-gateway/blob/develop/src/packages/db/db.ts#L28)

Reply:
It looks like the docs were incorrectly updated. I have fixed them 👍